Sitenotice.php

```diff
@@ -1,22 +1,22 @@
 <?php
 
-if ( $wmgSiteNoticeOptOut ) {
+ if ( $wmgSiteNoticeOptOut ) { */
```

Comment:
remove the */

Reply:
This was before your latest fix, I think?